src/atoms/Paragraph.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'react-emotion';
+import propTypes from 'prop-types';
+
+const P = styled.p`
+	font-size: 1.8rem;
```

Comment:
Did you consider adjusting BodyText instead of introducing this component?

Reply:
Also, this component should not hardcode font-size, font-weight, margin-bottom, but use theme variables  instead: uiRegularSize, uiWeight, uiWeightBold, verticalBase.